SPECS/php-sebastian-diff4/php-sebastian-diff4.spec

```diff
@@ -0,0 +1,200 @@
+# remirepo/fedora spec file for php-sebastian-diff4
+#
+# Copyright (c) 2013-2020 Remi Collet
+# License: CC-BY-SA
+# http://creativecommons.org/licenses/by-sa/4.0/
+#
+# Please, preserve the changelog entries
+#
+
+%bcond_without       tests
+
+# Namespace
+%global php_home     %{_datadir}/php
+
+Name:           php-sebastian-diff4
+Version:        4.0.4
+Release:        5%{?dist}
+Summary:        Diff implementation, version 4
+
+License:        BSD
+URL:            https://github.com/sebastianbergmann/diff
+Source0:        https://github.com/sebastianbergmann/diff/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#Source1:        makesrc.sh
+
+BuildArch:      noarch
+BuildRequires:  php-fedora-autoloader-devel
+%if %{with_check}
+BuildRequires:  php(language) >= 7.3
+BuildRequires:  php-pcre
+BuildRequires:  php-spl
+# from composer.json, "require-dev": {
+#        "phpunit/phpunit": "^9.3",
+#        "symfony/process": "^4.2 || ^5"
+BuildRequires:  phpunit9 >= 9.3
+BuildRequires:  php-symfony4-process
+%endif
+
+# from composer.json
+#        "php": ">=7.3"
+Requires:       php(language) >= 7.3
+# from phpcompatinfo report for version 4.0.0
+Requires:       php-pcre
+Requires:       php-spl
+# Autoloader
+Requires:       php-composer(fedora/autoloader)
+
+Provides:       php-composer(sebastian/diff) = %{version}
+
+
+%description
+Diff implementation.
+
+Autoloader: %{php_home}/SebastianBergmann/Diff4/autoload.php
+
+
+%prep
+%setup -q
+
+
+%build
+# Generate the Autoloader
+%{_bindir}/phpab --template fedora --output src/autoload.php src
+
+
+%install
+mkdir -p   %{buildroot}%{php_home}/SebastianBergmann
+cp -pr src %{buildroot}%{php_home}/SebastianBergmann/Diff4
+
+
+%check
+%if %{with_check}
```

Comment:
Please remove the if-else-end statement, this block will run only when tests are enabled